src/RoslynInsertionTool/RoslynInsertionTool/RoslynInsertionTool.VisualStudioTeamServices.cs

```diff
@@ -281,13 +281,14 @@ private static async Task<string> DownloadArtifactsAsync(BuildHttpClient buildCl
             if (Directory.Exists(tempDirectory))
             {
                 // Be judicious and clean up old artifacts so we do not eat up memory on the scheduler machine.
-                Directory.Delete(tempDirectory, recursive: true);
-                
-                // Sometimes a creation of a directory races with deletion since at least in .net 4.6 deletion is not a blocking call.
-                // Hence explictly waiting for the directory to be deleted before moving on.
-                Stopwatch w = Stopwatch.StartNew();
+                //Directory.Delete(tempDirectory, recursive: true);
 
-                while (Directory.Exists(tempDirectory) && w.ElapsedMilliseconds < 20 * 1000) Thread.Sleep(100);
+                //// Sometimes a creation of a directory races with deletion since at least in .net 4.6 deletion is not a blocking call.
```

Comment:
Comment cleanup.

Reply:
Sorry, this was not meant to be. I'll undo this change.